youtube_dl/extractor/wdr.py

```diff
@@ -21,54 +21,41 @@ class WDRIE(InfoExtractor):
 
     _TESTS = [
         {
-            'url': 'http://www1.wdr.de/mediathek/video/sendungen/servicezeit/videoservicezeit560-videoplayer_size-L.html',
+            'url': 'http://www1.wdr.de/mediathek/video/sendungen/hier_und_heute/videostreetfoodpioniere100.html',
```

Comment:
No. With `*-videoplayer_size-[LMS].html` a single media file URL is tested, otherwise - a playlist.